src/OSPSuite.UI/Views/Charts/DeviationLinesView.cs

```diff
@@ -0,0 +1,41 @@
+﻿using OSPSuite.Assets;
+using OSPSuite.Presentation.Extensions;
+using OSPSuite.Presentation.Presenters.Charts;
+using OSPSuite.Presentation.Views.Charts;
+using OSPSuite.UI.Extensions;
+using OSPSuite.Utility.Extensions;
+
+namespace OSPSuite.UI.Views.Charts
+{
+   public partial class DeviationLinesView : BaseModalView, IDeviationLinesView
+   {
+      private IDeviationLinesPresenter _presenter;
+      public DeviationLinesView()
+      {
+         InitializeComponent();
+         foldValueInputControlItem.TextVisible = false;
+         foldValueTextLabelControl.Text = Captions.Chart.DeviationLines.SpecifyFoldValue.FormatForLabel();
+         deviationLineDescriptionLabelControl.AsDescription();
+         deviationLineDescriptionLabelControl.Text = Captions.Chart.DeviationLines.DeviationLineDescription.FormatForDescription();
+         //Regex: only numbers, with 2 points of decimal precision, and greater than 1
```

Comment:
regex..oh no please :)
Let's create a DTO with a number and a rule and the screenbinder. This is how we do it everywhere. Right now you are bypassing the whole databinding and implementing everything by hand. why?

Reply:
used a DTO instead